lorien/InfiniteCanvas/Tools/CanvasTool.gd

```diff
@@ -38,7 +38,7 @@ func set_enabled(e: bool) -> void:
 	set_process_input(enabled)
 	_cursor.set_visible(enabled)
 	if enabled && _canvas:
-		_cursor.global_position = xform_vector2(get_viewport().get_mouse_position())
+		_cursor.global_position = xform_vector2(get_viewport().get_mouse_position()) * _canvas._scale
```

Comment:
Same here. Add a public `get_scale` to `InfiniteCanvas` and use that instead of `_canvas._scale`